src/ubuntu/20.04/helix/wasm/amd64/Dockerfile

```diff
@@ -1,59 +1,20 @@
-FROM ubuntu:20.04
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-amd64
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-# sources tweak is to work around very slow update
+# Need some additional packages
 
 RUN apt-get clean && \
```

Comment:
You'll need to add `USER root` before you can run package updates. Then at the end of the Dockerfile, you'll need to switch back: `USER helixbot`.